# Generated by Django 3.1.3 on 2020-11-07 13:26

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('complaint', '0001_initial'),
    ]

    operations = [
        migrations.CreateModel(
            name='Complaint_User',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('full_name', models.CharField(default=None, max_length=250)),
                ('email', models.CharField(default=None, max_length=250)),
                ('address', models.CharField(default=None, max_length=5000)),
                ('phone', models.CharField(blank=True, max_length=15, null=True)),
                ('alternative_phone', models.CharField(blank=True, max_length=15, null=True)),
                ('aadhar_no', models.CharField(default=None, max_length=15, unique=True)),
                ('pincode', models.CharField(default=None, max_length=10)),
                ('city', models.CharField(default=None, max_length=50)),
                ('state', models.CharField(default=None, max_length=50)),
                ('complaint_aadhar', models.ImageField(upload_to='complaint_aadhar')),
                ('complaint_image', models.ImageField(upload_to='complaint_image')),
                ('is_otp_verify', models.BooleanField(default=False)),
                ('otp_code', models.CharField(blank=True, max_length=6, null=True)),
                ('otp_created_at', models.DateTimeField(blank=True, null=True)),
                ('created_at', models.DateTimeField(auto_now_add=True)),
            ],
        ),
    ]
